SAGESharp/SLB/Common.cs

```diff
@@ -9,11 +9,13 @@
 using SAGESharp.Utils;
 using System;
 using System.Text;
+using SharpDX;
+using System.ComponentModel;
 
 namespace SAGESharp.SLB
 {
     /// <summary>
-    /// Class that represents an identifier within SLB files.
+    /// Immutable class that represents an identifier within SLB files.
```

Comment:
Minor, this is technically a `struct`

Reply:
Minor, disagree. Value type vs reference type has nothing to do with mutable vs immutable.

Reply:
Yeah, but the definition of `Identifier` is `struct`, that was what I meant

Reply:
Whoops, you're right